.travis.yml

```diff
@@ -33,13 +35,15 @@ env:
     - CONDA_CHANNEL_PRIORITY=True
     - NUMPY_VERSION=stable
 
+
+
 matrix:
   fast_finish: true
   include:
     - os : linux
       env: NAME='minimal'
            PYTHON_VERSION=2.7
-           SETUP_CMD='--with-memleak'
+           MAIN_CMD='python ./testsuite/MDAnalysisTests/mda_nosetests analysis auxiliary coordinates core formats topology --processes=2 --process-timeout=400 --no-open-files --with-timer --timer-top-n 50 --with-memleak && python ./testsuite/MDAnalysisTests/mda_nosetests utils --processes=2 --process-timeout=400 --no-open-files --with-timer --timer-top-n 50 --with-memleak'
```

Comment:
Yes we should use `&&`. Otherwise we are starting two test processes at the some time where each one uses the 2 processes internally. Giving a total of 4 running processes. But travis only gives us 2 cpu core. This means using anything but `&&` might hurt the runtime of the tests